cmd/init.go

```diff
@@ -48,18 +48,24 @@ func newInitCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// initCmd represents the init command
 	var initCmd = &cobra.Command{
 		Use:   "init [stack] or [repository]/[stack] [template]",
-		Short: "Initialize an Appsody project with a stack and template app",
-		Long: `This creates a new Appsody project in a local directory or sets up the local dev environment of an existing Appsody project.
-
-If the [repository] is not specified the default repository will be used. If no [template] is specified, the default template will be used.
-With the [stack], [repository]/[stack], [stack] [template] or [repository]/[stack] [template] arguments, this command will setup a new Appsody project. It will create an Appsody stack config file, unzip a template app, and run the stack init script to setup the local dev environment. It is typically run on an empty directory and may fail
-if files already exist. See the --overwrite and --no-template options for more details.
-Use 'appsody list' to see the available stack options.
-
-If keyword "none" is specified instead of a [template], the project will be initialized to use Appsody, and no template will be provided.
-
-Without the [stack] argument, this command must be run on an existing Appsody project and will only run the stack init script to
-setup the local dev environment.`,
+		Short: "Initialize an Appsody project.",
+		Long: `Sets up the local Appsody development environment around an existing project or, optionally provides a template project. In all cases, this command unzips a template application, and runs the stack init script to setup the local development environment. 
```

Comment:
`In all cases, this command unzips a template application, and runs the stack init script to setup the local development environment.`

I don't believe this is accurate.

Reply:
How about:

Sets up the local Appsody development environment. You can do this for an existing project or create a new template application.

Reply:
Should be "Set up...", to be consistent with the "create" command, and others. 
Think the other sentence needs a bit of clarification - "Set up the local Appsody development environment for an existing project" makes sense, but "Set up the local Appsody development environment create a new template application" doesn't.